pkg/autodiscovery/autoconfig.go

```diff
@@ -454,6 +411,55 @@ func (ac *AutoConfig) pollConfigs() {
 	}()
 }
 
+// TODO: move to check scheduler
```

Comment:
I am not sure we should remove this piece of code to the check Scheduler, we should rather call the `MetaScheduler` here no ?

Reply:
Yep but this specific piece of code will be moved to the check scheduler